src/main/java/com/ystrazhko/git/ui/javafx/LoginWindow.java

```diff
@@ -66,14 +66,20 @@ public void handle(ActionEvent e) {
                 actiontarget.setFill(Color.FIREBRICK);
                 String name = userTextField.getText();
                 String password = pwBox.getText();
-
+                try {
                 Object json = getLoginService().login(name, password);
-                actiontarget.setText(String.valueOf(json != null));
+                actiontarget.setText("Successful connection");
+
                 //debug code
                 ((GroupsUserService) ServiceProvider.getInstance().getService
                         (GroupsUserService.class.getName())).getGroups(json.toString());
 
-                System.out.println();
+                } catch (Exception exception) {
+                    System.err.println("!ERROR: " + exception.getMessage());
+                    String message = exception.getMessage();
+                    message = message.substring(message.indexOf("HTTP"), message.indexOf(" for"));

```

Comment:
That's bad info for user. I am seeing in login window: HTTP response code: 401. I think that message should be more clear. Something like: "Invalid credantionals"